packages/aragon-wrapper/package-lock.json

```diff
@@ -1,8 +1,6 @@
 {
-	"name": "@aragon/wrapper",
```

Comment:
@AquiGorka to be honest i'm not sure why there are changes in `package-lock.json`. I made a mistake and deleted the original one and so i copied this one from master. So don't why it shows the diffs.

Reply:
Ok. You probably just got a different revision from the file, maybe try again with previous ones: https://github.com/aragon/aragon.js/commits/master/package-lock.json

Reply:
Ok thanks will do

Reply:
Thank you.
If I can offer another suggestion: try to keep your commits independent/atomic (as much as possible because some changes do modify more than one file), this will allow you to revert/remove commits by doing an [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)

Reply:
@AquiGorka ahhh ok, got it. Will do so from now on 🙂